flink-docs/src/main/java/org/apache/flink/docs/configuration/ConfigOptionsDocGenerator.java

```diff
@@ -66,6 +66,7 @@
 		new OptionsClassLocation("flink-yarn", "org.apache.flink.yarn.configuration"),
 		new OptionsClassLocation("flink-mesos", "org.apache.flink.mesos.configuration"),
 		new OptionsClassLocation("flink-mesos", "org.apache.flink.mesos.runtime.clusterframework"),
+		new OptionsClassLocation("flink-metrics/flink-metrics-influxdb", "org.apache.flink.metrics.influxdb"),
```

Comment:
The InfluxDbOptions aren't picked up by the generator on purpose since they clash with the prometheus options, and the generator can't differentiate between them yet.

Reply:
so this stuff and related changes should be reverted